redaxo/src/core/pages/setup.php

```diff
@@ -195,6 +195,9 @@
         }
     } elseif (3 == $createdb) {
         $import_name = rex_post('import_name', 'string');
+        /**  @psalm-taint-escape text */
+        $import_name = basename($import_name);
```

Comment:
```suggestion
        $import_name = rex_path::basename($import_name);
```

Ich weiß nicht genau, ob hier nötig. Aber an anderer Stelle hatten wir das Problem, dass basename nicht reichte, weil wir nachträglich noch Slashes/Backslashes umgewandelt haben.
Deswegen hatten wir die Methode eingeführt.
Siehe https://github.com/redaxo/redaxo/pull/1579/files

Reply:
hmm sollten wir ggf. via SA verbieten dass man `basename` aufrufen darf?

Reply:
Weiß nicht so genau, ob ich es wirklich global verbieten würde.
Andererseits gibt es vermutlich keine Stelle, wo es wirklich schadet, unsere eigene Methode zu verwenden. Teilweise wird es höchstens unnötig sein.

Reply:
ok, separates thema: https://github.com/redaxo/redaxo/issues/4069